src/Tizen.Wearable.CircularUI.Forms.Renderer/ContentButtonRenderer.cs

```diff
@@ -0,0 +1,156 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using ElmSharp;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+using Xamarin.Forms.Platform.Tizen.Native;
+using Tizen.Wearable.CircularUI.Forms;
+using Tizen.Wearable.CircularUI.Forms.Renderer;
+using XForms = Xamarin.Forms.Forms;
+using EBox = ElmSharp.Box;
+using EButton = ElmSharp.Button;
+using EColor = ElmSharp.Color;
+
+[assembly: ExportRenderer(typeof(ContentButton), typeof(ContentButtonRenderer))]
+
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class ContentButtonRenderer : VisualElementRenderer<ContentButton>
```

Comment:
Is it better to use `VisualElementRenderer` than to use `ViewRenderer`?

Reply:
Fixed